url/src/comp.php

```diff
@@ -0,0 +1,477 @@
+<?php
+
+print("</br><a href='url.html'>Back to url tool</a>");
+
+// ini_set('display_errors', 1);
+// ini_set('display_startup_errors', 1);
+ini_set('max_execution_time', '600');
+// error_reporting(E_ALL);
+
+echo "<p>Starting...</p>";
+
+// User set variables
+// Get the user config file. This script will fail disgracefully if it has not been created and nothing will happen.
+require('../../user.config.php');
+
+echo "Tenancy Shortcode set: " . $shortCode;
+echo "</br>";
+
+echo "Client ID set: " . $clientID;
+echo "</br>";
+
+echo "User GUID to use: " . $TalisGUID;
+echo "</br>";
+
+// Test run or live run switch
+if(isset($_REQUEST['DRY_RUN']) &&
+	$_REQUEST['DRY_RUN'] == "writeToLive") {
```

Comment:
One more thing - Recommend ``===`` here for comparison too.  Same reason as in the previous comment that if ``$_REQUEST['DRY_RUN']`` is ``0`` then this will evaluate to true.

Reply:
updated. Thank you